doc/tsc-meetings/2015-08-26.md

```diff
@@ -0,0 +1,110 @@
+Target ETA for transition: Monday.
+
+
+### doc: merge CHANGELOG.md with joyent/node ChangeLog [#2536](https://github.com/nodejs/node/pull/2536)
+
+* Rod seeking confirmation from this group that this was _not a bad idea_.
+* Discussion about documentation for people upgrading from older versions of Node and having a linear history. Jeremiah has been working on 0.10 -> v4 documentation.
+
+### (guidance on) rename of "language groups" from iojs-* to nodejs-*? [#2525](https://github.com/nodejs/node/issues/2525)
+
+* Steven discussed renaming the language groups
+* Mikeal: some groups are not in a position to make decisions because they are not active but the ones that are should be given the opportunity to agree or disagree to the move rather than us imposing a move on them. We should post an issue in their repos.
+* Steven: we should give them a timeline in issues in their repos
+* Discussed naming, could be “lang” or “community” to make it more clear what they are about.
+* ACTION: Steven to add a proposal to #2525, let TSC/collaborators add comments, later in the week will reach out to each of the groups.
+
+### Node.js v4 Release Timeline [#2522](https://github.com/nodejs/node/issues/2522)
+
+* Rod gave a summary of the proposed release timeline https://github.com/nodejs/node/issues/2522 (and appologised for documenting it as “the” release proposal just for the sake of getting it done). 
+  - “feature freeze” Friday, 28th of August, cut v4.x branch
+  - nodejs.org DNS changeover to new website Monday, 31st of August
+  - release Thursday, 3rd of September
+  - RC builds between website and release
+  - backup release date of Monday, 7th of September if absolutely necessary to punt
+* Discussed the outstanding items in the 4.0.0 milestone: https://github.com/nodejs/node/milestones/4.0.0, some concerning items:
+  - mdb support is late and may not get done, might have to be a semver-minor prior to LTS
+  - vcbuilt.bat needs to use the new build_release target and we have to verify that all builds have Intl enabled - Steven to work with Rod on this
+  - `process.send()` async/sync, Ben said that this is async on all platforms now (was just Windows previously but now it’s cross-platform) but it’s missing a callback so there’s no way of doing back-pressure. No time to add this prior to v4 but could be done as semiver-minor later. Ben agreed to document current behaviour properly in the docs.
+  - `_unrefActive` is in Jeremiah’s hands, he will land the 0.12 changes as the io.js version is terrible. Discussed further perf improvements but agreed to leave those changes till later, ideally till v5 because of the potential for subtle edge-case bugs being introduced.
+
+### srl295 - how to onboard into WG? https://github.com/nodejs/collaboration/issues/4
```

Comment:
Title is very long. It can be written that way:  `[nodejs/collaboration#4](https://github.com/nodejs/collaboration/issues/4)`